assets/src/edit-story/components/canvas/singleSelectionMoveable/index.js

```diff
@@ -65,6 +65,16 @@ function SingleSelectionMoveable({
     backgroundElement: currentPage.elements[0] ?? {},
   }));
 
+  useEffect(() => {
+    const handleUpdateMoveable = () => {
+      moveable.current?.updateRect();
+    };
+    window.addEventListener('resize', handleUpdateMoveable);
+    return () => {
+      window.removeEventListener('resize', handleUpdateMoveable);
+    };
+  }, [moveable]);
```

Comment:
We could also move it to a reusable hook: `useResizeableMovable(ref)` or similar?

Reply:
Hm, wasn't certain if it's worth it in this case but will move if you think it's better!